src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -174,6 +171,93 @@ class SidebarLinks extends React.PureComponent {
         this.props.onLinkClick();
     }
 
+    showSettingsPage() {
+        if (this.props.isCreateMenuOpen) {
+            // Prevent opening Settings page when click profile avatar quickly after clicking FAB icon
+            return;
+        }
+
+        Navigation.navigate(ROUTES.SETTINGS);
+    }
+
+    showStatusPage() {
+        if (this.props.isCreateMenuOpen) {
+            // Prevent opening Settings page when click profile avatar quickly after clicking FAB icon
+            return;
+        }
+
+        Navigation.setShouldPopAllStateOnUP();
+        Navigation.navigate(ROUTES.SETTINGS_STATUS);
+    }
+
+    renderAvatarWithIndicator() {
```

Comment:
I would extract this method to the separate functional component

Reply:
👍🏻